qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -18,10 +18,6 @@
 from shutil import copyfile
 
 import numpy as np
-from openfermion import MolecularData, QubitOperator
-from openfermion.transforms import bravyi_kitaev, get_fermion_operator, jordan_wigner
-from openfermionpsi4 import run_psi4
-from openfermionpyscf import run_pyscf
```

Comment:
@agran2018 it's my understanding that `run_psi4` and `run_pyscf` should be imported within the scf driver functions, but the other OpenFermion imports should be okay to leave at the top of the file